### ISSUE TITLE
[feature] UX: click-through relations

### ISSUE THREAD

**Issue by heymartinadams** [prisma/studio#499]
1. Prisma2 version (`prisma -v`):

A lot of our schema is tightly interconnected. Many queries are deeply nested. For example, given a certain `eventId`, we’ll need to look up if a user is a participant: event → event option → registrations → participants → users

In code, this could look something like this:
```js
const users = await prisma.user.findMany({
  where: {
    AND: [
      { id },
      {
        participants: {
          some: {
            registrations: {
              some: {
                eventOption: {
                  eventId: {
                    equals: eventId
                  }
                }
              }
            }
          }
        }
      }
    ]
  }
})
```

Part of Prisma’s beauty is our ability to accomplish exactly that: finding results based on deeply nested relations.

Once we’ll be in production, however, our customer service team will need to perform similar queries, using Studio. However, the UI doesn’t really permit them to look up deeply nested relations.

For example, in the `User` table, I’m looking up when a user was a `Participant`. From there, I’d like to see what `Registration`s the `Participant` was a part of (and, from there, what `EventOption` they registered for, and then what `Event` those `EventOption`s were a part of) — however, at this point, we cannot click through any further relations (see below):

![ezgif](https://user-images.githubusercontent.com/11673745/90590736-29273200-e196-11ea-9ebe-f39c1826bfff.gif)

Instead, at this point, we need to write down the relevant `Participation` ids, and then open a new tab and explore the relations further, which is quite cumbersome.

I wonder if there’s a UX that allows for an infinite click-through experience. Hope that makes sense.

**Comment by madebysid** [comment]
@heymartinadams There's now a new `Open in New Tab` button in the relation table that should let you accomplish this.
What this button does is open a new tab with appropriate filters applied so the rows you see will be the connected records.

So in your case, clicking on the button in the relation table from your GIF would open a new tab with the `SystemLogs` table with two rows visible: the one with `participantID` `par192d...` & `par_6ea9...`

Let me know how that works out. It is available currently in `@prisma/cli@2.6.0-dev.54` and above. This will also land in the 2.6.0 release!

**Comment by heymartinadams** [comment]
That works really well, thanks everyone!
